### PR TITLE
Mobile version of the Personal Portfolio Project

### PR DESCRIPTION
**Pull request by Jhaman1895**
## In this Milestone, I have taken the following steps to set up the "navbar" and "hero" section of the mobile version of the "Personal Portfolio" project: 

- [ ] Created a new GitHub repo.
- [ ] Added a `.gitignore` file.
- [ ] Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config).
- [ ] Made sure that any files that are not relevant are ignored by git.
- [ ] Added a descriptive README file.
- [ ] Used HTML to structure the skeleton of the page.
- [ ] Used CSS to style the page
- [ ] Implemented CSS Flexbox to create the layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <div class="welcome-container">
+            <!--Header Begins-->
+            <div class="toolbar-container">
+                <header class="toolbar">
+                    <div class="logo-container">
+                        <a href="#" class="logo">Ayush Jha</a>
+                    </div>
+                    <div class="menu-container">
+                        <a href="#" class="menu"><img src="./assets/images/Union.svg" alt="" class="menu-image"></a>
+                    </div>
```
Comment (awais-amjed) [inline index.html]: Kindly change this `div` to a `nav` tag 😃

- _The `nav` tag is for semantics reasons. Later in the project, you will add a navigation menu that opens when you click the hamburger._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                </header>
+            </div>    
+            <!--Header Ends-->           
+            <!--Hero Section Begins-->
+                <div class="hero-container">
```
Comment (awais-amjed) [inline index.html]: Kindly change this `div` tag to a `section` tag for semantic reasons 😀

- _A Section represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info._

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @Jhaman1895 , 

Good job so far! 👏👏 Awesome Design 🥳🎈🧨
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter errors ✔️ 
- [x] Nice and Responsive Design 👏👏
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [x] Kindly note that the background image needs some adjusting as you can see from the images below 😄

_Your Design_
image

_Figma Design_
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jhaman1895** [comment]
Fixed the background image and changed divs to semantic tags as per the code-reviewers suggestion.

**Review by phelian23 — APPROVED** [review]
Hi @Jhaman1895,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Great job setting up the correct linters ✔️
Professional documentation ✔️
Correct workflow ✔️
Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**